specs/checkVersion.spec

```diff
@@ -0,0 +1,19 @@
+PIPELINES-18
+# Specification Heading
+
+## Scenarion for testing pipeline version
+
+tags: e2e
+
+This is the a comment for the scenario
+we are passing a spec table for the verification.
+
+* Verify component versions 
+
+   |component |Version    |
+   |----------|-----------|
```

Comment:
@ppitonak, would it make sense to have these values in `pkg/config.go` instead of here as these versions keep on changing for each release and we cannot have a single test pointing to multiple versions?